deploy/examples/create-external-cluster-resources.py

```diff
@@ -827,45 +819,96 @@ def gen_json_out(self):
                 }
             }
         ]
-
-        # if 'ROOK_EXTERNAL_DASHBOARD_LINK' exists, then only add 'rook-ceph-dashboard-link' Secret
-        if self.out_map['ROOK_EXTERNAL_DASHBOARD_LINK']:
-            json_out.append({
-                "name": "rook-ceph-dashboard-link",
-                "kind": "Secret",
-                "data": {
-                    "userID": 'ceph-dashboard-link',
-                    "userKey": self.out_map['ROOK_EXTERNAL_DASHBOARD_LINK']
-                }
-            })
-        # if 'CSI_RBD_PROVISIONER_SECRET' exists, then only add 'rook-csi-rbd-provisioner' Secret
-        if self.out_map['CSI_RBD_PROVISIONER_SECRET']:
+        if self._arg_parser.restricted_auth_permission:
+            cluster_name = self._arg_parser.cluster_name
+            rbd_pool_name = self._arg_parser.rbd_data_pool_name
+            cephfs_filesystem = self._arg_parser.cephfs_filesystem_name
+            namespace = self._arg_parser.namespace
             json_out.append({
-                "name": "rook-csi-rbd-provisioner",
+                "name": "rook-csi-rbd-node-{}-{}-{}".format(cluster_name,rbd_pool_name,namespace),
```

Comment:
How to get the rados namespace for the specific pool, currently just using the cluster namespace?